bedrock/firefox/templates/firefox/products/desktop.html

```diff
@@ -0,0 +1,155 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/products/product-base.html" %}
+
+{# L10n: strings for this page are not considered final and aren't read for localization. #}
+
+{% block page_title_prefix %}{% endblock %}
+{% block page_title %}{{ _('Fast, private, modern browser for desktop computers | Firefox') }}{% endblock %}
+{% block page_title_suffix %}{% endblock %}
+{% block page_desc %}{{ _('Firefox is a faster, safer and more private browser than ever before — for Windows, Mac, and Linux desktop computers. Download now!') }}{% endblock %}
+{% block page_image %}{{ static('img/firefox/products/desktop/product-desktop-meta.jpg') }}{% endblock %}
+
+{% block body_id %}firefox-product-desktop{% endblock %}
+
+
+{% block product_header_content %}
+  <h1 class="page-title"><span>{{ _('Firefox for Desktop') }}</span></h1>
+  <p class="tagline">{{ _('Get more done — faster with Firefox.') }}</p>
+{% endblock %}
+
+{% block product_header_download %}
+  {{ download_firefox(dom_id='product-header-download', button_color='button-hollow-arrow dark') }}
+{% endblock %}
+
+{% block product_header_image %}
+  {{ high_res_img('firefox/products/desktop/product-desktop-hero.jpg', {'class': 'hero-image', 'alt': '', 'width': '700', 'height': '390'}) }}
+{% endblock %}
+
+
+{% block features_list %}
+<section id="features" class="section features">
+  <div class="content">
+
+    <h2 class="section-title"><span>{{ _('Features') }}</span></h2>
+
+    <ul class="features-list">
+      <li class="features-list-item">
+        <a href="#TODO-/features/modern" data-link-type="link" data-link-position="Feature 1" data-link-name="Get more done">
+          <img class="feature-image" src="{{ static('img/firefox/products/desktop/feature-desktop-speed.jpg') }}" alt="">
+        </a>
+        <h3 class="feature-title">{{ _('Get more done') }}</h3>
+        <div class="feature-desc">
+          <p>
+          {% trans %}
+            Now Firefox is even faster – with quicker start ups and less hang time!
+            That means windows and tabs load quicker for lightweight browsing that
+            doesn’t hog your computer’s memory.
+          {% endtrans %}
+          </p>
+        </div>
+        <p class="cta"><a href="#TODO-/features/modern" class="cta-link" data-link-type="link" data-link-position="Feature 1" data-link-name="Get more done">{{ _('Get more done at Firefox speeds') }}</a></p>
+      </li>
+
+      <li class="features-list-item">
+        <a href="{{ url('firefox.private-browsing') }}" data-link-type="link" data-link-position="Feature 2" data-link-name="Get more privacy">
```

Comment:
Yeah, this is a link that will need updating when your branch lands. I'll replace it with a `#TODO` so we don't miss it (and I'll replace all the other `#` links with `#TODO` as well, for easier searching).